src/city/ratings.c

```diff
@@ -316,90 +317,37 @@ static void update_culture_rating(void)
-    }
-    city_data.ratings.culture += city_data.ratings.culture_points.theater;
-
-    int pct_religion = city_data.culture.religion_coverage;
-    if (pct_religion >= 100) {
-        city_data.ratings.culture_points.religion = 30;
-    } else if (pct_religion > 85) {
-        city_data.ratings.culture_points.religion = 22;
-    } else if (pct_religion > 70) {
-        city_data.ratings.culture_points.religion = 14;
-    } else if (pct_religion > 50) {
-        city_data.ratings.culture_points.religion = 9;
-    } else if (pct_religion > 30) {
-        city_data.ratings.culture_points.religion = 3;
-    } else {
-        city_data.ratings.culture_points.religion = 0;
-    }
-    city_data.ratings.culture += city_data.ratings.culture_points.religion;
-
-    int pct_school = city_culture_coverage_school();
-    if (pct_school >= 100) {
-        city_data.ratings.culture_points.school = 15;
-    } else if (pct_school > 85) {
-        city_data.ratings.culture_points.school = 10;
-    } else if (pct_school > 70) {
-        city_data.ratings.culture_points.school = 6;
-    } else if (pct_school > 50) {
-        city_data.ratings.culture_points.school = 4;
-    } else if (pct_school > 30) {
-        city_data.ratings.culture_points.school = 1;
-    } else {
-        city_data.ratings.culture_points.school = 0;
-    }
-    city_data.ratings.culture += city_data.ratings.culture_points.school;
-
-    int pct_academy = city_culture_coverage_academy();
-    if (pct_academy >= 100) {
-        city_data.ratings.culture_points.academy = 10;
-    } else if (pct_academy > 85) {
-        city_data.ratings.culture_points.academy = 7;
-    } else if (pct_academy > 70) {
-        city_data.ratings.culture_points.academy = 4;
-    } else if (pct_academy > 50) {
-        city_data.ratings.culture_points.academy = 2;
-    } else if (pct_academy > 30) {
-        city_data.ratings.culture_points.academy = 1;
-    } else {
-        city_data.ratings.culture_points.academy = 0;
-    }
-    city_data.ratings.culture += city_data.ratings.culture_points.academy;
-
-    int pct_library = city_culture_coverage_library();
-    if (pct_library >= 100) {
-        city_data.ratings.culture_points.library = 20;
-    } else if (pct_library > 85) {
-        city_data.ratings.culture_points.library = 14;
-    } else if (pct_library > 70) {
-        city_data.ratings.culture_points.library = 8;
-    } else if (pct_library > 50) {
-        city_data.ratings.culture_points.library = 4;
-    } else if (pct_library > 30) {
-        city_data.ratings.culture_points.library = 2;
-    } else {
-        city_data.ratings.culture_points.library = 0;
-    }
-    city_data.ratings.culture += city_data.ratings.culture_points.library;
-
+	
+    //ENTERTAINMENT: max 25
+    city_data.ratings.culture_points.average_entertainment_cp = (city_data.culture.average_theater + city_data.culture.average_amphitheater + city_data.culture.average_colosseum + city_data.culture.average_hippodrome) / 16;
```

Comment:
Magic numbers are never a good idea. I managed to understand what that `16` meant, which is "the max averaged sum is `400` and we want it to be `25`", but it's not immediately obvious.

Maybe you chould do something like this:

```
int average_entertainment = (city_data.culture.average_theater + city_data.culture.average_amphitheater + city_data.culture.average_colosseum + city_data.culture.average_hippodrome) / 4;

// ENTERTAINMENT: max 25
city_data.ratings.culture_points.average_entertainment_cp = average_entertainment / 4;
```

The same applies to education.